### ISSUE TITLE
hevea -fix tutoriel.tex: /bin/bash: line 1: hevea: command not found

### ISSUE THREAD

**Issue by mkoeppe** [passagemath/giac#1]
https://github.com/passagemath/giac/actions/runs/11569379770/job/32203201708#step:11:4135

**Comment by mkoeppe** [comment]
@parisseb @orlitzky Following up on a previous discussion in https://github.com/sagemath/sage/pull/35745, here is a new repository from which I hope to generate "upstream tarballs" suitable for the Sage distribution. It is using Debian's patches on top of giac version 1.9.0.93; for now I have not applied any of the old patches that Sage carries.

After a few edits, I am able to run `make dist` successfully. 
The first problem I'm running into when building from the source distribution built this way is that it wants to run `hevea` which is not available. Could this step be made conditional?

**Comment by parisseb** [comment]
hevea is called to build the online doc of Giac/Xcas from latex source files, but this should not happen if the doc files from the tar archive were not modified, this is controlled by a stamp file. When I run make dist myself, hevea is not called. 
Perhaps I missed to add some stamp files in the archive and this should be modified in a Makefile.am somewhere. Which tex file was/were passed as argument to hevea?

**Comment by mkoeppe** [comment]
> Which tex file was/were passed as argument to hevea?

tutoriel.tex

**Comment by mkoeppe** [comment]
@parisseb Here hevea was called both during `make dist` (https://github.com/passagemath/giac/actions/runs/11569379770/job/32202982138) and then again when building from the tar archive that was created by `make dist`.

**Comment by parisseb** [comment]
Indeed, I checked and found 2 missing stamps for tutoriel in fr and en, this should be fixed in the next tarball, thanks!

**Comment by mkoeppe** [comment]
Great, thanks! For now I have patched out "tutoriel" in my build to see if there are other issues.

**Comment by mkoeppe** [comment]
@parisseb I had to make some other minor adjustments for `make dist` to go through here. If you have a moment, please take a look at https://github.com/passagemath/giac/commit/fa023ce07fdb01e95190f16c20d022d40d34f116 to see if this should be applied on your side. (I have not checked yet if some of it is due to Debian's modifications of the sources.)

**Comment by parisseb** [comment]
I don't think so. Almost all files that you removed in EXTRA_DIST are useful for me for building giac on some platforms, like windows mingw, OSX or calculators. And anyway, these files are small, removing the files that are really not useful anymore would not change much the tarball size...
